maven-plugin/src/main/java/org/openmrs/maven/plugins/utility/OwaHelper.java

```diff
@@ -300,7 +304,7 @@ private String runProcessAndGetFirstResponseLine(String command, @Nullable Strin
 			lines = IOUtils.readLines(process.getInputStream(), StandardCharsets.UTF_8);
 			process.waitFor();
 		} catch (InterruptedException | IOException e) {
-			System.out.println(e.getMessage());
+			logger.error(e.getMessage());
```

Comment:
It is usually a good practice to pass the exception object as the second parameter.